docs/how-to-guides/faq.md

```diff
@@ -84,6 +84,22 @@ Currently, promptflow supports the following environment variables:
 
 Valid for batch run only. The number of workers to use for parallel execution of the Flow.
 
+Default value is 16. If you want more efficiency, you can increase the number to the improve the batch run concurrency, make it run faster.
+
+If your batch run fails due to rate limit of your LLM endpoint, pls set up PF_WORKER_COUNT to a smaller number. Take Azure OpenAI endpoint as example, you can go to Azure OpenAI Studio, navigate to Deployment tab, check out the capacity of your endpoints. Then you can refer to this expression to set up the concurrency.
+
+```
+PF_WORKER_COUNT <= TPM * duration_seconds / token_count / 60
+```
+TPM: token per minute, capacity rate limit of your LLM endpoint
+
+duration_seconds: single flow run duration en seconds
+
+token_count: single flow run token count
+
+
+For example, if your endpoint TPM (token per minute) is 40K, the single flow run takes 10k tokens and runs for 30s, pls do not set up PF_WORKER_COUNT bigger than 2. This is a rough estimation. Please also consider collboaration (teammates use the same endpoint at the same time) and tokens consumed in deployed inference endpoints, playground and other cases which might send request to your LLM endpoints.
```

Comment:
If one single flow run takes 10k token and runs for 30s, and PF_WORKER_COUNT is set to 2, then pf can finish 4 inputs in 1 minutes, which means `10K * 4 = 40K` tokens in one minute, exactly hit the limit of the endpoint. If PF_WORKER_COUNT set to more than 2 (e.g. 3), then 60K tokens would be used in one minute, and RateLimitError would be raised since it is larger than the TPM.

Reply:
ok, I see your point, we should not take a worker number exactly reaching the limit, we need to be slightly smaller than it. I'll change the TPM to 50k, it that okay?